spec/descriptions/tagInfrastructureAnalyze.md

```diff
@@ -0,0 +1,45 @@
+The following endpoints expose our analyze functionality.
+Two endpoints allow to [list entity types](#operation/getAvailablePlugins) and [metrics for an entity type](#operation/getAvailableMetrics)
+It is possible to [search and filter entities](#operation/getEntities) and even to [aggregate metrics on similar entities](#operation/getEntityGroups)
+
+## Common Parameters
+
+### Mandatory Parameters
+
+**timeFrame** As in our UI you can specify the timeframe for metrics retrieval.
+```
+  windowSize           to
+     (ms)       (unix-timestamp)
+<----------------------|
+```
+The timeFrame might be adjusted to fit the metric granularity so that there is no partial bucket. For example, if the query timeFrame is 08:02 - 09:02 and the metric granularity is 5 minutes, the timeFrame will be adjusted to 08:05 - 09:00. The adjusted timeFrame will be returned in the response payload. If the query does not have any metric with granularity, a default granularity will be used for adjustment.
+
+### Optional Parameters
+
+**pagination**
+* *offset* set the starting point for the data retrieval
+* *retrievalSize* you set the number of returned values
+* *ingestionTime* if you want to paginate through your result set you are interested in having the data for a fixed time point, the results set has a `cursor` class that has a ingestionTime property that indicates what you have to enter here.
+
+**order**
+* *by* metric name or `label`
```

Comment:
Since _direction_ is an enum in the API docs, I figured it did not need further explanation, though I agree it looks like it's missing. I wonder if adding description directly on the field would work instead.